sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapCameraManagerDelegate.kt

```diff
@@ -0,0 +1,286 @@
+package com.mapbox.maps.plugin.delegates
+
+import com.mapbox.geojson.Geometry
+import com.mapbox.geojson.Point
+import com.mapbox.maps.*
+
+/**
+ * Definition of a camera delegate. Any invocation will interact with the map's actual camera.
+ */
+interface MapCameraManagerDelegate {
+  /**
+   * Get current latitude.
+   * @return latitude
+   */
+  fun getLat(): Double
+
+  /**
+   * Get current longitude.
+   * @return longitude
+   */
+  fun getLon(): Double
```

Comment:
why do we separate `getLat` and `getLon`, is there any specific use case? otherwise can we expose something like `getCenter` which will return a point?

Reply:
I'm not sure, these are just copied from old `MapCameraDelegate.kt`

Reply:
Just a nit, maybe worth ticketing it out as a tailwork. We need to review these interfaces.

Reply:
Ticketed in https://github.com/mapbox/mapbox-maps-android-internal/issues/354